.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  generate-winmd:
+    name: Generate winmd
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      # No precompiled winmd checked in
+      # - name: Clean
+      #   run: Remove-Item -Recurse -Force .windows/winmd/
```

Comment:
This'll have to be removed as @jenatali mentioned in https://github.com/microsoft/DirectX-Headers/pull/82#issuecomment-1322177524 that we should keep `.winmd` just in CI artifacts (and attach them to GH releases/tags), and not check them in.